helix_fhir_client_sdk/responses/fhir_client_protocol.py

```diff
@@ -134,15 +132,6 @@ def page_size(self, page_size: int) -> "FhirClientProtocol": ...
 
     def filter(self, filter_: List[BaseFilter]) -> "FhirClientProtocol": ...
 
-    async def get_by_query_in_pages_async(
-        self,
-        concurrent_requests: int,
-        output_queue: asyncio.Queue[PagingResult],
-        fn_handle_batch: Optional[HandleBatchFunction],
-        fn_handle_error: Optional[HandleErrorFunction],
-        fn_handle_streaming_chunk: Optional[HandleStreamingChunkFunction],
-    ) -> AsyncGenerator[FhirGetResponse, None]: ...
```

Comment:
why is this method not needed any more? 
is there any other alternative method taking care of the action already??